docs/README.md

```diff
@@ -1,114 +1,232 @@
 # ScalarDB Benchmarks
 
-This repository contains benchmark programs for ScalarDB.
+This tutorial describes how to set up and run benchmarking tools for ScalarDB.
```

Comment:
I'd like to mention the significance of benchmarking here. How about we add something like the following?:

```suggestion
This tutorial describes how to run benchmarking tools for ScalarDB. Database benchmarking is helpful for evaluating how databases perform against a set of standards.
```